builders/build/eth-api/dev-env/openzeppelin/defender.md

```diff
@@ -26,118 +29,93 @@ OpenZeppelin Defender现已上线Moonbeam、Moonriver和Moonbase Alpha测试网
 
 本教程使用的合约是在[智能合约升级教程](https://docs.openzeppelin.com/learn/upgrading-smart-contracts){target=\_blank}中的`Box.sol`合约的拓展，是OpenZeppelin文档中抽取的合约。此外，本合约可升级、[可暂停](https://docs.openzeppelin.com/contracts/4.x/api/security#Pausable){target=\_blank}，能够充分发挥Admin组件的优势。您可以使用以下代码进行合约部署，并根据[智能合约升级教程](https://docs.openzeppelin.com/learn/upgrading-smart-contracts){target=\_blank}进行操作：
 
 ```solidity
-// SPDX-License-Identifier: MIT
-pragma solidity ^0.8.0;
-
-import "@openzeppelin/contracts-upgradeable/security/PausableUpgradeable.sol";
-import "@openzeppelin/contracts-upgradeable/access/OwnableUpgradeable.sol";
-
-contract PausableBox is Initializable, PausableUpgradeable, OwnableUpgradeable {
-    uint256 private value;
- 
-    // Emitted when the stored value changes
-    event ValueChanged(uint256 newValue);
-
-    // Initialize
-    function initialize() initializer public {
-        __Ownable_init();
-        __Pausable_init_unchained();
-    }
- 
-    // Stores a new value in the contract
-    function store(uint256 newValue) whenNotPaused public {
-        value = newValue;
-        emit ValueChanged(newValue);
-    }
- 
-    // Reads the last stored value
-    function retrieve() public view returns (uint256) {
-        return value;
-    }
-    
-    function pause() public onlyOwner {
-        _pause();
-    }
-
-    function unpause() public onlyOwner {
-        _unpause();
-    }
-}
+--8<-- 'code/builders/build/eth-api/dev-env/openzeppelin/PausableBox.sol'
 ```
 
-### 连接Defender {: #connecting-defender }
+!!! 注意事项
+    使用Remix或其他工具（例如Hardhat）部署完上述合约后，您需要调用 `initialize` 函数来正确设置可升级合约的所有者。如果您不调用此函数，所有
 
-注册了OpenZeppelin Defender账户以后，登录[Defender App](https://defender.openzeppelin.com/){target=\_blank}。在主屏幕中，您将看到[MetaMask已连接到Moonbase Alpha](/builders/get-started/networks/moonbase/#connect-metamask){target=\_blank}，点击右上角的**Connect wallet**（连接钱包）按钮：
+## 使用Access Control组件 {: #using-the-access-control-component }
 
-![OpenZeppelin Defender Connect](/images/builders/build/eth-api/dev-env/openzeppelin/defender/oz-defender-1.webp)
+本节将引导您使用[OpenZeppelin Defender的访问控制（Access Control）组件](https://defender.openzeppelin.com/v2/#/access-control/contracts){target=_blank} 在Moonbase Alpha上管理智能合约。
 
-如果操作成功，您可以看到自己的地址，并显示**Connected to Moonbase Alpha**（已连接到Moonbase Alpha）。
+### 导入合约 {: #importing-your-contract }
 
-## 使用Admin组件 {: #using-the-admin-component }
+使用Defender访问控制的第一步是添加您想要管理的合约。请按照以下步骤操作：
 
-本小节将介绍如何使用OpenZeppelin Defender Admin组件管理Moonbase Alpha上的智能合约。
+ 1. 点击 **Access Control** 菜单项 
+ 2. 点击 **Add Contract**
+ 3. 为您的合约添加一个名称
+ 4. 选择部署合约的 **Network** 。这个演示中我们选择Moonbase Alpha
+ 5. 粘贴合约地址
+ 6. 如果您的合约已被验证，它的ABI将自动导入，否则需要在这粘贴合约ABI。合约ABI可以在[Remix](https://remix.ethereum.org/){target=_blank}中获取，也可以在编译过程后（例如Hardhat）创建的 `.json` 文件中找到
+ 7. 检查完所有信息后，单击 **Create** 按钮
 
-### 导入合约 {: #importing-your-contract }
+ ![OpenZeppelin Defender Access Control Add Contract](/images/builders/build/eth-api/dev-env/openzeppelin/defender/new/oz-defender-1.png)
```

Comment:
```suggestion
 ![OpenZeppelin Defender Access Control Add Contract](/images/builders/build/eth-api/dev-env/openzeppelin/defender/oz-defender-1.webp)
```

So in translations like this one, the original PR says that the images need to be updated after translations. So, you'll need to open a PR on the `moonbeam-docs` repo that removes the images directly in the `oz-defender` directory and replace them with the ones in the `oz-defender/new` directory - so that there is no longer a `new` directory and all of the new images replace the old ones. You'll also need to update the `defender.md` page so that the image paths no longer have `new` in them

Reply:
please make sure to update all of the image paths in this PR to  remove `new` and switch from `.png` to `.webp`